moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -88,6 +81,17 @@ const DEFAULT_HIDDEN_COLS = {
   public_process_status: true,
 };
 
+/**
+ * Keeps localStorage column config in sync with UI interactions
+ * @param {Object} column - the MT column config with the `field` prop - aka the column name
+ * @param {Bool} hidden - the hidden state of the column
+ */
+const handleColumnChange = ({ field }, hidden) => {
+  let storedConfig = JSON.parse(localStorage.getItem("mopedColumnConfig"));
+  storedConfig = { ...storedConfig, [field]: hidden };
+  localStorage.setItem("mopedColumnConfig", JSON.stringify(storedConfig));
```

Comment:
at first glance it looked to me like this would break if the user did not already have `mopedColumnConfig` in local storage, but `getItem` will return `null` if the item is not found, and `JSON.parse(null)` returns `null`, and `{...null}` returns `{}`. ✨